alcs-api/src/import/import.service.ts

```diff
@@ -0,0 +1,383 @@
+  Kootenay: 'koot',
+  Okanagan: 'okan',
+  'South Coast': 'south',
+};
+
+const TYPE_MAPPING = {
+  INC: 'INCL',
+  SDV: 'SUBD',
+  EXC: 'EXCL',
+  NFU: 'NFUP',
+  TUR: 'TURP',
+  NAR: 'NARU',
+  FILL: 'POFO',
+  EXT: 'ROSO',
+  SCH: 'PFRS',
+  CSC: '',
+  SRW: '',
+};
+
+@Injectable()
+export class ImportService {
+  private logger: Logger = new Logger(ImportService.name);
+
+  constructor(
+    private applicationService: ApplicationService,
+    private meetingService: ApplicationMeetingService,
+    private pausedService: ApplicationPausedService,
+    private boardService: BoardService,
+    private localGovernmentService: ApplicationLocalGovernmentService,
+  ) {}
+
+  importCsv() {
+    this.logger.log('Import Setup Started');
+    return new Promise(async (resolve, reject) => {
+      const mapping = await this.loadMappingSheet();
+
+      const filePath = path.resolve(__dirname, '..', 'Tracking_Sheet.csv');
+      const stream = fs.createReadStream(filePath);
+      const processing = [];
+      let i = 0;
+      stream
+        .pipe(csv())
+        .on('data', (data) => {
+          if (data['Decision Released (Active)']) {
+            return;
+          }
+
+          const promise = this.parseRow(data, mapping);
+          processing.push(promise);
+          i++;
+        })
+        .on('end', () => {
+          Promise.all(processing).then(() => {
+            this.logger.log(`Processed ${processing.length} Records`);
+            resolve('Complete');
+          });
+        });
+    });
+  }
+
+  async parseRow(
+    data: ApplicationRow,
+    mapping: Map<string, { localGovernment: string; type: string }>,
+  ) {
+    const mappedRow = this.mapRow(data);
+
+    const existingApplication = await this.applicationService.get(
+      mappedRow.fileNumber,
+    );
+    if (existingApplication) {
+      this.logger.debug(`${mappedRow.fileNumber}: Application already exists`);
+    } else {
+      try {
+        const sheetType = mapping.get(mappedRow.fileNumber);
+        if (!sheetType) {
+          this.logger.warn(`${mappedRow.fileNumber}: Not in mapping sheet`);
+          return;
+        }
+
+        const mappedType = TYPE_MAPPING[sheetType.type];
+        if (!mappedType) {
+          this.logger.warn(
+            `${mappedRow.fileNumber}: Non-importable type ${sheetType.type}`,
+          );
+          return;
+        }
+
+        let localGovernment = await this.localGovernmentService.getByName(
+          sheetType.localGovernment,
+        );
+        if (!localGovernment) {
+          localGovernment = await this.localGovernmentService.getByName(
+            `${sheetType.localGovernment} Regional District`,
+          );
+          if (!localGovernment) {
+            this.logger.error(
+              `${mappedRow.fileNumber}: Failed to load government ${sheetType.localGovernment}`,
+            );
+            return;
+          }
+        }
+
+        const application = await this.applicationService.create({
```

Comment:
So this is going to create an application in DB for every single record from csv?